doc/installing.rst

```diff
@@ -40,6 +40,15 @@ The package and version requirements for ``PypeIt`` are:
 * `sphinx-automodapi <https://pypi.org/project/sphinx-automodapi/>`_ : version 0.12 or later; optional, developers only
 * `sphinx_rtd_theme <https://pypi.org/project/sphinx-rtd-theme/>`_ : version 0.5 or later; optional, developers only
 
+Developer-only items
+--------------------
+
+If you are developing, you may need the following packages:
+
+* `sphinx <https://www.sphinx-doc.org/en/master/>`_ -- version 4.0 or later
+* sphinx_automodapi (pip install only)
+* sphinx_rtd_theme (pip install only)
+
```

Comment:
Doesn't this repeat the information that's above?

Reply:
yeah, must have been the  merge.  I knew I had 
added these previously.  ;)

edited